contracts/SpokePool.sol

```diff
@@ -578,6 +664,67 @@ abstract contract SpokePool is
      * @param message The message to send to the recipient on the destination chain if the recipient is a contract.
      * If the message is not empty, the recipient contract must implement handleV3AcrossMessage() or the fill will revert.
      */
+    function depositV3Now(
+        bytes32 depositor,
+        bytes32 recipient,
+        bytes32 inputToken,
```

Comment:
Interface design question: if we _know_ this token is local (and therefore an address), is there a point in making it bytes32?

Same question for depositor.

There may be an advantage in keeping them consistent, but there is a disadvantage in validation/error cases: it's possible to pass in `bytes32` addresses that do not correspond to a valid local address.

Reply:
The main reason was consistency with the rest of the data. We still have the [overloaded](https://github.com/across-protocol/contracts/blob/d97325003a1a38828bb7b87b52be67c3f25ab65a/contracts/SpokePool.sol#L728-L740) versions of these functions that can help with the validation/casting of addresses.

We have tried to move everything to bytes32 in this contract. While in the short term there might be a bit of friction, I think in the medium term it’s better to have everything in either bytes32 or address, but not mixed, as mixing is more error-prone. However, we have made these changes somewhat radically and are looking for your feedback.

We do have bytes32 address encoding checks https://github.com/across-protocol/contracts/blob/d97325003a1a38828bb7b87b52be67c3f25ab65a/contracts/libraries/AddressConverters.sol#L11-L13 to partially help with the address to bytes32 conversion.